rust/processor/src/processors/token_v2_processor.rs

```diff
@@ -1127,6 +1188,7 @@ async fn parse_v2_token(
                 &b.storage_id,
             ))
     });
+    current_token_royalties_v1.sort();
```

Comment:
Does it automatically know to sort by token_data_id (pk)?

Reply:
yea cuz I added the sort function remember. that was your idea.